src/app/components/mno-app-features/mno-app-features.html

```diff
@@ -0,0 +1,20 @@
+<div id="app-features">
+  <div class="btn-group">
+    <div for="frm-test-elm-100-1" class="btn btn-default">
+      <span class="glyphicon" ng-class="isSingleSignOn ? 'fa fa-check-square-o fa-lg' : 'fa fa-square-o fa-lg'"></span>
+      <span class="content">{{ 'mno_enterprise.templates.app_feature.single_sign_on' | translate }}</span>
+    </div>
+  </div>
+  <div class="btn-group top-space">
+    <div for="frm-test-elm-100-1" class="btn btn-default">
```

Comment:
Also, why do you use a button class here?
It changes style when you click on it, it is only displayed to the user.